android/app/src/main/java/com/moviedbdemo/SplashActivity.java

```diff
@@ -0,0 +1,17 @@
+package com.moviedbdemo; // Replace this with your package name
+
+import android.content.Intent;
+import android.os.Bundle;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+public class SplashActivity extends AppCompatActivity {
```

Comment:
No need to create SplashActivity, you can do it directly instead of this. so please check it and remove the splash activity code.